pkg/tool/helm.go

```diff
@@ -18,6 +18,8 @@ import (
 	"fmt"
 
 	"github.com/helm/chart-testing/v3/pkg/exec"
+
+	"strings"
```

Comment:
can you move this to close to the "fmt" part

should be like

```
import (
	"fmt"
	"strings"

	"github.com/helm/chart-testing/v3/pkg/exec"
)
```

Reply:
Done